cli/tsc/mod.rs

```diff
@@ -106,7 +106,8 @@ pub fn get_types_declaration_file_text(unstable: bool) -> String {
 }
 
 fn get_asset_texts_from_new_runtime() -> Result<Vec<AssetText>, AnyError> {
-  // the assets are stored within the typescript isolate, so take them out of there
+  // the assets are stored within the typescript isolate, so take them out of
+  // there
```

Comment:
"Let me remove the word "the" at the start to prevent wrapping" - Me if I had wrote this comment once this lands (not necessarily a bad thing)